flycheck.el

```diff
@@ -4315,8 +4329,11 @@ default `:verify' function of command checkers."
         (parser (or (plist-get properties :error-parser)
                     #'flycheck-parse-with-patterns))
         (predicate (plist-get properties :predicate))
-        (standard-input (plist-get properties :standard-input)))
-
+        (standard-input (plist-get properties :standard-input))
+        (cwd       (plist-get properties :cwd)))
+    ;; FIXME add more to the cwd checker
+    ;; (unless (stringp (car cwd))
+    ;;   (error "Current working directory must be a string"))
```

Comment:
ups. I need to still update this to check if the property is either a string (fixed path) or a function.